python/ray/_private/structured_logging/utils.py

```diff
@@ -0,0 +1,44 @@
+from ray._private.structured_logging.formatters import LogfmtFormatter
+from ray._private.structured_logging.filters import CoreContextFilter
+
+# A dictionary to map encoding types to their corresponding logging configurations.
+LOG_MODE_DICT = {
+    "TEXT": {
+        "version": 1,
```

Comment:
TODO: I will investigate the meaning for `version` and `disable_existing_loggers`.